pyproject.toml

```diff
@@ -1,16 +1,15 @@
 [tool.pyright]
 
 # add any files/directories with type declaration to include
-include = ["pettingzoo/utils/env.py"]
+include = [
+    "pettingzoo/utils/env.py"
```

Comment:
For now we only want type checking for that single file?

I think it'd be better to include everything by default, and add explicit includes for each directory/file/env/whatever, so that new code is also type checked

Reply:
If this is too big of a job, I'm more than ok with merging this PR first, and then starting new PRs for each of the other folders.

Reply:
Yea, my point is changing the structure of include/exclude.

When done this way, let's say someone develops a new feature for PZ, puts it in a new file or directory or anywhere really, and pushes changes there. None of these changes will be type checked in any way.

Reply:
Since https://github.com/Farama-Foundation/PettingZoo/pull/737/commits/c07a637a3d568044172bd074a87ad880981da8d1#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711 it'll type check any file in the `pettingzoo` folder excluding the list of folders that haven't been refactored with typing yet. I also added types to files in `pettingzoo/utils/`. Does this work?